about.html

```diff
@@ -0,0 +1,204 @@
+          
+        </p>
+        <p class="font-2 color-2 desktop-only">
+          I Was Honored To Be Part Of The Second Annual Conference,
+          Presenting To An Enthusiastic Crowd Of Women And Men Committed To Making IWC A Shining Example Of Equality And Empowerment
+        </p>
+        <span class="cc-logo">
+          <img src="images/women-conference-Logo.jpg" alt="CC logo" />
+        </span>
+      </section>
+      <section id="events" class="past-events">
+        <h3 class="font-2 color-2 sub-header">
+          Previous International women's Conference
+        </h3>
+        <div class="red-line"></div>
+        <p class="color-2 font-2">
+          Take a look at the last two CC Summits that took place in Nigeria and
+          South Africa
+        </p>
+        <div class="previous-events">
+          <div class="e-2019 event">
+            <h4 class="font-2">2019</h4>
+            <p class="font-2">
+              International Women's Conference in Abuja, Nigeria
+            </p>
+          </div>
+          <div class="e-2021 event">
+            <h4 class="font-2">2021</h4>
+            <p class="font-2">
+              International Women's Conference in Pretoria, South Africa
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="partners mobile-only">
+        <h6 class="font-2 color-3">Partners</h6>
+        <div class="red-line"></div>
+        <div class="partner-logo">
+          <img
+            class=""
+            src="../images/partners/nike-logo.svg"
+            alt="nike logo"
+          />
+          <img
+            class=""
+            src="../images/partners/addidas.png"
+            alt="adiddas logo"
+          />
+          <img
+            class=""
+            src="../images/partners/google-logo.svg"
+            alt="google logo"
+          />
+          <img
+            class=""
+            src="../images/partners/mozilla.svg"
+            alt="mozilla logo"
+          />
+          <img class="" src="../images/partners/gucci.svg" alt="gucci logo" />
+        </div>
+      </section>
+      <footer>
+        <span class="footer-logo mobile-only">
+          <img src="images/women-conference-Logo.jpg" alt="CC Logo" />
+        </span>
+        <span class="footer-logo desktop-only">
+          <img src="images/women-conference-Logo.jpg" alt="CC Logo" />
+        </span>
+        <div class="footer-text">
+          <p class="font-2 color-2">
+            2022 International Women's Conference. All Rights Reserved.
+          </p>
+          <p class="desktop-only font-2 color-2">
+            All contents on this website, unless stated otherwise, is at the
+            International Women's Conference.
+            <br />For detailed usage, function details and permission
+            conditions, see http://iwc.org
+          </p>
+        </div>
```

Comment:
- [ ] Please place right and left margin to your footer content, It should be in center as on the design


### Your design

![image](https://user-images.githubusercontent.com/83803414/158951899-8970055a-a085-44be-8ac9-4d91f91dd4be.png)

### Behance design
![image](https://user-images.githubusercontent.com/83803414/158951862-cf96c5a1-4890-4396-835b-f378f0d174b0.png)